### PR TITLE
[#114474167] Feature/organizer edit page

### PR DESCRIPTION
**Pull request by bigsis**

### CODE REVIEW

**Comment by bigsis** [comment]
screen shot 2559-03-07 at 7 36 14 pm
screen shot 2559-03-07 at 7 36 28 pm

**Comment by bigsis** [comment]
screen shot 2559-03-08 at 3 32 44 pm

File: app/assets/stylesheets/backend/application.css.scss [hunk 1]
```diff
@@ -27,6 +27,7 @@
 @import './module/image_upload';
 @import './module/logo';
 @import './module/devise_form';
+@import './module/organizer_container';
```
Comment (aunsuwijak) [inline app/assets/stylesheets/backend/application.css.scss]: ```'./module/organizer_settings';```

File: app/assets/stylesheets/backend/module/_organizer_container.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.organizer-container {
```
Comment (aunsuwijak) [inline app/assets/stylesheets/backend/module/_organizer_container.scss]: ```.organizer-settings-container ```

File: backend/actions/organizer-actions.jsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import dispatch from '../dispatch.jsx';
```
Comment (aunsuwijak) [inline backend/actions/organizer-actions.jsx]: indent by 2 space

File: backend/app-constant.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export default {
```
Comment (aunsuwijak) [inline backend/app-constant.jsx]: indent by 2 space

File: backend/components/interface-component.jsx [hunk 5]
```diff
@@ -4,6 +4,7 @@ import EventListContainer from './events/list-container.jsx';
 import EventCreateContainer from './events/create-container.jsx';
 import EventShowContainer from './events/show-container.jsx';
 import EventEditContainer from './events/edit-container.jsx';
+import ContentContainer from './organizers/content-container.jsx';
```
Comment (aunsuwijak) [inline backend/components/interface-component.jsx]: SettingContainer

File: backend/components/interface-component.jsx [hunk 6]
```diff
@@ -64,6 +65,15 @@ class InterfaceComponent extends React.Component {
         </div>
       );
     }
+
+    if(this.props.router.current === 'organizers') {
```
Comment (aunsuwijak) [inline backend/components/interface-component.jsx]: ``` if (this.props.router.current === 'organizers/settings') ```

File: app/assets/stylesheets/backend/module/_organizer_container.scss [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      margin-top: $grid-gutter-width;
+    }
+  }
+
+
+  @include media-breakpoint-up(md) {
+    width: $grid-gutter-width*24;
+    padding: $grid-gutter-width;
+    margin: $grid-gutter-width auto 0;
+    background-color: rgba(255, 255, 255, 0.5);
```
Comment (aunsuwijak) [inline app/assets/stylesheets/backend/module/_organizer_container.scss]: check this color in hex

File: backend/router.jsx [hunk 8]
```diff
@@ -28,6 +30,14 @@ var Router = Backbone.Router.extend({
   editEvent: function(id) {
     this.current = 'events/edit';
     this.params = { _id: id };
+  },
+
+  organizers: function() {
+    this.current = 'organizers';
+  },
+
+  organizersUpdate: function() {
```
Comment (aunsuwijak) [inline backend/router.jsx]: remove this route

File: backend/stores/organizer-store.jsx [hunk 9]
```diff
@@ -0,0 +1,51 @@
+import Backbone from 'backbone';
```
Comment (aunsuwijak) [inline backend/stores/organizer-store.jsx]: indent with 2 space

File: backend/stores/organizer-store.jsx [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+            .fetch({
+                url: `/api/v1/organizers/${payload.organizer.id}`,
+                data: formData,
+                type: 'PUT',
+                processData: false,
+                contentType: false
+            });
+
+        jqXHR.done(() => {
+            //Backbone.history.navigate('app/organizers', true);
```
Comment (aunsuwijak) [inline backend/stores/organizer-store.jsx]: remove comment

**Pull request merged**